crates/ty_python_semantic/resources/mdtest/snapshots/mro.md_-_Method_Resolution_Or…_-_`__bases__`_lists_wi…_(ea7ebc83ec359b54).snap

```diff
@@ -141,7 +141,7 @@ info[revealed-type]: Revealed type
 ```
 
 ```
-error[duplicate-base]: Duplicate base class `Eggs`
+error[duplicate-base]: Duplicate base class `Spam`
```

Comment:
Not sure why this snapshot here changed. *Surely*, it has nothing to do with non-deterministic normalization of unions.

Reply:
Probably because the iteration order of that hashmap here is not deterministic?
```rs
let mut base_to_indices: FxHashMap<ClassBase<'db>, Vec<usize>>
```

Reply:
Oh, that's interesting. Maybe we should be using an `IndexSet` there. The order changed in https://github.com/astral-sh/ruff/pull/18060 as well.